src/app/controllers/PokemonController.js

```diff
@@ -0,0 +1,72 @@
+import Pokemon from '../models/pokemon';
+import paginationHandler from '../services/paginationHandler';
+
+class PokemonController {
+  async getInfo(req, res) {
+    const info = [];
+    info.push({
```

Comment:
Como os dados passados pro info são fixos, poderia fazer tudo em um comando só: 
```suggestion
    const info = [{
      description: 'Banco de dados com informações referentes ao jogo Pokemon Go!',
      tableSize: await Pokemon.count(),
      pokemonAttributes: await Pokemon.getAttributes(),
    }]
```

Reply:
Isso eu também tinha mudado, mesmo problema lá dos scripts do banco de dados, vou acertar aqui tb.